### ISSUE TITLE
Add option output FTP URL

### ISSUE THREAD

**Issue by Arttse** [timomeh/filezilla-sitemanager-cli#4]
I need to get ftp url, like `ftp://user:password@host:port/path`. Can you do this option?

**Comment by timomeh** [comment]
Yep, seems like a helpful thing. I sure can add it, if you're patient until around 1st April. I'm currently kinda on vacation. If you want to, you could also create a PR.
